src/Codeception/Module/REST.php

```diff
@@ -333,6 +333,18 @@ public function amNTLMAuthenticated($username, $password)
     }
 
     /**
+     * Adds AWS Authentication via key, secret, service and region
+     * @param $key
+     * @param $secret
+     * @param $service
+     * @param $region
+     */
+    public function amAWSAuthenticated($key, $secret, $service, $region)
+    {
+        $this->client->setAwsAuth($key, $secret, $service, $region);
```

Comment:
Please check if the setAwsAuth method exists before calling it. Guzzle isn't the only client used by REST module.

Reply:
Cleaned it up